grasa_event_locator/forms.py

```diff
@@ -23,7 +23,7 @@
     ("Public Speaking", "Public Speaking"),
     ("Social and Emotional Learning (SEL)", "Social and Emotional Learning (SEL)"),
     ("Sports and Recreation", "Sports and Recreation"),
-    ("STEM", "STEM"),
+    ("Science, Tech, Engineering, Math (STEM)", "Science, Tech, Engineering, Math (STEM)"),
```

Comment:
is this strictly necessary?

Reply:
@nlMeminger Yes. The acronym is not always equally understood. It's also already hard-coded in the back-end so any time the filter is used in `master` without this PR, the event editor breaks.